Animal Competition/Java/Game.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+class Game {
+  void startGame(){
+    String greeting = "Hello! Welcome to (insert name here)!";
+
+    System.out.println(greeting);
+
+    Scanner playerNumber = new Scanner(System.in);
+
+    System.out.println("Press 1 for singleplayer. Press 2 for two players.");
+    
+    int numberOfPlayers = playerNumber.nextInt();
+      // if one player is playing
+    if (numberOfPlayers == 1) {
+      System.out.println("Welcome!");
+      
+      // makes a new scanner to scan for user input
+      Scanner userNameInput = new Scanner(System.in);
+      
+    // prompts user for username
+      System.out.println("Please enter your Username:");
+
+      String userName = userNameInput.nextLine();
+    // welcome user with their desired username
+      System.out.println("Welcome " + userName + "!");
+
+      // if 2 players are playing
+    } else if (numberOfPlayers == 2){
+      
+      System.out.println("Welcome!");
+      
+      System.out.println("Player 1 please enter a username:");
```

Comment:
And you can reuse the `getPlayerName` function here:
```suggestion
      var user1Name = this.getPlayerName("Player 1");
      var user2Name = this.getPlayerName("Player 2");
      System.out.println(String.format("Welcome %s and %s", user1Name, user2Name));
```